### DISCUSSION TITLE
Updating Acs commons package AEM Cloud as a Service.

### DISCUSSION THREAD

**Discussion by okmi** [Adobe-Consulting-Services/acs-aem-commons#3343] [General]
## Required Information

* [ ] AEM Version, including Service Packs, Cumulative Fix Packs, etc: 2024.4.16145.20240430T082417Z
* [ ] ACS AEM Commons Version: 4.9.2

Hello we have been asked from our client to update the acs commons package in our AEM as a cloud server environments, from 4.9.2 version to the latest 6.5.0 and although i imagine this should be an easy transition i was wondering if we should have any concerns about the transition and compatibility to a new version if something could break.

Any insight or things to look out for would be greatly appreciated it.

We are getting this error log while trying to upload it through compilation in our environment:

Failed to execute goal on project parques-reunidos.all: Could not resolve dependencies for project com.parques.reunidos:parques-reunidos.all:content-package:1.0-SNAPSHOT: The following artifacts could not be resolved: com.adobe.acs:acs-aem-commons-ui.content:zip:min:6.5.0 (absent), com.adobe.acs:acs-aem-commons-ui.apps:zip:min:6.5.0 (absent): com.adobe.acs:acs-aem-commons-ui.content:zip:min:6.5.0 was not found in
https://repo.maven.apache.org/maven2
during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of central has elapsed or updates are forced

**Comment by kwin** [comment]
Please consider https://adobe-consulting-services.github.io/acs-aem-commons/pages/maven.html for the slightly changed Maven coordinates
